### PR TITLE
Adding note about office hours, removing note about developer meetings

### PR DESCRIPTION
**Pull request by arschles**
**What is the problem I am trying to address?**

[docs.gomods.io](https://docs.gomods.io) and the README still reference the developer meetings, but we haven't done those in a few months, and it's unlikely that we'll resume them soon.

**How is the fix applied?**

I've replaced the developer meetings page (and all links to it) with an office hours page.

**Mention the issue number it fixes or add the details of the changes if it doesn't have a specific issue.**

N/A

@LinuxSuRen @lopezator would you mind checking out the Chinese / Spanish documentation (respectively) and changing the wording in there, when you can?

### CODE REVIEW

File: docs/content/contributing/community/office-hours.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Developer Meetings"
```
Comment (robjloranger) [inline docs/content/contributing/community/office-hours.md]: Should the title change to 'Office Hours' as well?

**Comment by lopezator** [comment]
@arschles the term "office hours" is an standard term for this kind of meetings or something relative to the athens project? Not sure how to translate this to spanish, if it's a custom term maybe the literal translation could work...

**Comment by robjloranger** [comment]
@lopezator it's a standard concept where an expert makes themselves available for questions. Typically it was something that professors do, so students had time outside of lectures to get further help or understanding.

It has been adopted by tech experts and projects to do something similar.

Someone correct me if in wrong, but in this case it is part code walkthrough and part office hours.

Hope that helps.

**Comment by lopezator** [comment]
@robjloranger thanks! I get it. I think I have now the correct term to translate this into spanish, thanks!

File: docs/content/contributing/community/office-hours.md (hunk 1 above)
Comment (arschles) [reply docs/content/contributing/community/office-hours.md]: Yes! Good catch, changing now

File: docs/content/contributing/community/office-hours.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+title: "Office Hours"
+date: 2018-09-28T10:40:50-07:00
+weight: 3
+---
+
+Athens developers get together for an hour or so approximately every week on a video chat to learn and discuss a part of the codebase. The hour is relatively unstructured, but generally it starts out with a core maintainer (usually [@arschles](https://github.com/arschles)) going into the details of a part of the codebase. From there, we just go where the questions take us.
+
+This is a great opportunity for new community members to get more involved with the project in a low-pressure setting. It's also a great setting for existing community members to gain a deeper understanding into the codebase.
+
+We meet at this [zoom](https://zoom.us/) video Chat URL: [https://arschles.com/zoom](https://arschles.com/zoom)
```
Comment (jpreese) [inline docs/content/contributing/community/office-hours.md]: The capital C in `Chat` seems a little out of place to me

File: docs/content/contributing/community/office-hours.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+>**Absolutely everybody is welcome to attend these meetings. You're free to suggest topics to talk about, share your perspective, and participate as little or much as you'd like**.
+>
+>You don't have to be a contributor to attend.
+
+# Logistics
+
+All office hours happen using the [zoom](https://zoom.us) software. Before you come to the meeting, please make sure you have installed the zoom client using this link: https://zoom.us/support/download.
```
Comment (jpreese) [inline docs/content/contributing/community/office-hours.md]: We should probably wrap the `/download` link in markdown

File: docs/content/contributing/community/office-hours.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Office Hours"
+date: 2018-09-28T10:40:50-07:00
```
Comment (jpreese) [inline docs/content/contributing/community/office-hours.md]: Do we want the date to be consistent with the old developers-meeting file? As it's a new topic, we may want to consider updating

File: docs/content/contributing/community/office-hours.md (hunk 3 above)
Comment (arschles) [reply docs/content/contributing/community/office-hours.md]: @jpreese after a long delay, I updated this 😆 . Can you let me know what you think?

Comment (jpreese) [reply docs/content/contributing/community/office-hours.md]: No worries! LGTM :)

**Comment by arschles** [comment]
CI/CD on drone stays pending forever on this PR for some reason, I'm going to merge anyway because this is docs only

**Comment by jpreese** [comment]
Yeah man, no worries. I didn't see anything else noteworthy.

**Pull request merged**